filters/security/filter_42_ShadowWhisperersMalwareList/metadata.json

```
@@ -0,0 +1,14 @@
+{
+    "filterId": "shadowwhisperers_malware_list",
+    "id": 42,
+    "name": "ShadowWhisperer's Malware List",
+    "description": "Malicious Sites, PUPs, Malware, Brower Hijackers, Phishing Sites",
+    "timeAdded": 1681400345355,
+    "homepage": "https://github.com/ShadowWhisperer/BlockLists/blob/master/Lists/Malware",
```

the home page of the repo

```suggestion
    "homepage": "https://github.com/ShadowWhisperer/BlockLists",
```